src/content/docs/develop/calling-rust.mdx

```diff
@@ -47,6 +47,35 @@ const invoke = window.__TAURI__.invoke;
 invoke('my_custom_command');
 ```
 
+If you are using rust-based frontend and want to use `invoke()` without arguments,
+because rust doesn't support optional arguments,
+you will need to change code on your frontend from this:
+
+```rust
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(js_namespace = ["window", "__TAURI__", "core"])]
+    async fn invoke(cmd: &str, args: JsValue) -> JsValue;
+}
+```
+
+to this
+
+```rust
+#[wasm_bindgen]
+extern "C" {
+    // invoke without arguments
+    #[wasm_bindgen(js_namespace = ["window", "__TAURI__", "core"], js_name = invoke)]
+    async fn invoke_without_args(cmd: &str) -> JsValue;
+
+    // invoke with arguments (default)
+    #[wasm_bindgen(js_namespace = ["window", "__TAURI__", "core"])]
+    async fn invoke(cmd: &str, args: JsValue) -> JsValue;
+
+    // They need to have different names!
+}
+```
```

Comment:
Do you mind adding text markers https://expressive-code.com/key-features/text-markers/ so it can be written in only one code block? 

expressive code is already part of starlight, I'm referring to add something like `ins={"3":10-12}` to highlight added and changed lines